app/controllers/hub/efile_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Hub
+  class EfileController < Hub::BaseController
+    include FilesConcern
+    authorize_resource
+    load_resource except: [:index, :show]
+    layout "hub"
+    def index
+      @efile_submissions = EfileSubmission.includes(:efile_submission_transitions, tax_return: [:client, :intake]).most_recent_by_current_year_tax_return.page(params[:page])
+      @efile_submissions = @efile_submissions.in_state(params[:status]) if params[:status].present?
+    end
+
+    def show
```

Comment:
our State-file intakes do not have clients or tax returns unlike CTC where we basing this page off of  -- not sure if the plan is to add tax returns or to change the model so they `has_one :intake` not `through: :tax_return`. But I'm guessing in any case that this show page should be using the StateFile intake models instead of the client

Reply:
Done